R/use-bbi.R

```diff
@@ -18,40 +22,60 @@ use_bbi <- function(.dir = "/data/apps"){
 
   this_os <- os[sapply(os,grepl,x=R.version$os)]
 
+  if(.version == "latest") {
+    .bbi_url <- current_release(owner = 'metrumresearchgroup', repo = 'babylon', os = "linux")
```

Comment:
I changed this, but if it's not Linux, `.bbi_url` just gets thrown away. We should probably clean this up, but let's do it in the refactor branch because this function is probably gonna get changed a lot there anyway.